auth-manager/src/main/java/com/qpay/authmanager/service/jwt/impl/JwtServiceImpl.java

```diff
@@ -34,6 +34,7 @@ public String generateToken(final TokenData tokenData) {
     private String createToken(final Map<String, Object> claims) {
         return Jwts.builder()
                 .setClaims(claims)
+                .setSubject(claims.get("email").toString())
```

Comment:
why did use decide move it back?

Reply:
https://github.com/Hoocki/qpay/pull/90#issue-2266958905